homeworks/boroczpeter/2_vars_datatypes/fictive_character extra.py

```diff
@@ -0,0 +1,20 @@
+#Second homework, task 1+extra
+
+#asking for character informations
+name = input ("Please give me your name: ").strip().upper()
+age_years = int(input("How old are you? "))
+python_exp_in_years = int(input("Your Python programming experience in years? "))
+ #question of extra task
+professional_developer = input("Do you want to be a profesional Python developer? (yes/no) ").strip().lower()
+
+#calculate character age in days (asuming a year with 365 days )
+age_in_days = age_years*365 #created a new variable
+
+#using ternary operator in Boolean variable
```

Comment:
Mi lenne, ha a ternary operator nem egy True-t vagy False-t adna vissza, hanem azt a 2 string részletet ami eltér a 2 kiprintelt mondatod között?
"wants" és "does not want". 
Ezt a két értéket vehetné fel a want_to_be_dev változód (vagy ha más nevet adsz neki emiatt, akkor az), és elég lenne egyszer printelned úgy, hogy ezt a változót beágyazod a mondatban a He/she után. Így nem kéne még egy if-else.